### DISCUSSION TITLE
Copy recipe json files in published folder

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#8496] [Q&A]
Hi,
Is it possible to copy recipe json files in published folder?
I would like to edit a recipe file after deploying OC application.

**Comment by jtkech** [comment]
Modules recipes are embedded static files that you can't update, we would need a new feature to create / edit recipes through the admin. That said under the admin configuration there are import / export / deployment plan features, where you can export / import json files, with a plan you can define a given set of pre-defined recipe steps to be exported, you can also define remote servers / clients. There is also a JSON import page where you can edit json data, then by clicking on import it creates a temporary json file and executes it. May help to search infos in the docs https://docs.orchardcore.net/en/dev/

**Reply by DotCat1985** [reply]
Ok. Thank you
